api/serverV2_test.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/testutil"
+)
+
+func TestServerV2Start(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, _ := createServerV2(cfg, false)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	err := svr.Start()
+	require.NoError(err)
```

Comment:
should start() end stop() without no error

Reply:
yes